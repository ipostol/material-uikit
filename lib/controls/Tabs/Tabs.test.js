'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _enzyme = require('enzyme');

var _Tabs = require('./Tabs');

var _Tabs2 = _interopRequireDefault(_Tabs);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('Tabs component', function () {

  it('should render correctly', function () {

    var props = {
      tabs: [{ id: 1 }],
      onChange: function onChange() {}
    };

    var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(_Tabs2.default, props));

    wrapper.find('div.tab').simulate('click');

    expect(wrapper).toMatchSnapshot();
  });
});