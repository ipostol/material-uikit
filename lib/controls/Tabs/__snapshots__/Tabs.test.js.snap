// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs component should render correctly 1`] = `
<div
  className="general"
>
  <div
    className="tab"
    onClick={[Function]}
  />
  <div
    className="underline"
  />
</div>
`;
